os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_scatternet/ble_tizenrt_scatternet_app.c

```diff
@@ -699,12 +699,12 @@ void ble_tizenrt_scatternet_app_handle_dev_state_evt(T_GAP_DEV_STATE new_state,
     APP_PRINT_INFO3("ble_tizenrt_scatternet_app_handle_dev_state_evt: init state  %d, scan state %d, cause 0x%x",
                     new_state.gap_init_state,
                     new_state.gap_scan_state, cause);
-    dbg("init state  %d, scan state %d, conn_state %d, adv_state %d, cause 0x%x \n",
-                    new_state.gap_init_state,
-                    new_state.gap_scan_state,
-                    new_state.gap_conn_state,
-                    new_state.gap_adv_state,
-                    cause);
+    debug_print("init state  %d, scan state %d, conn_state %d, adv_state %d, cause 0x%x \n",
+                    new_state.gap_init_state,
+                    new_state.gap_scan_state,
+                    new_state.gap_conn_state,
+                    new_state.gap_adv_state,
+                    cause);

```

Comment:
```suggestion
    dbg("init state  %d, scan state %d, conn_state %d, adv_state %d, cause 0x%x \n",
                    new_state.gap_init_state,
                    new_state.gap_scan_state,
                    new_state.gap_conn_state,
                    new_state.gap_adv_state,
                    cause);
```

Reply:
@giwon-nam the modification is made accordingly